tests/e2e/test_e2e.py

```diff
@@ -364,10 +364,10 @@ def test_e2e(caplog):
     L_train_gold = labeler.get_gold_labels(train_cands, annotator="gold")
     assert L_train_gold[0].shape == (3493, 1)
 
-    gen_model = LabelModel(k=2)
-    gen_model.train_model(L_train[0], n_epochs=500, print_every=100)
+    gen_model = LabelModel()
+    gen_model.fit(L_train=L_train[0].toarray(), n_epochs=500, log_freq=100)
 
-    train_marginals = gen_model.predict_proba(L_train[0])
+    train_marginals = gen_model.predict_proba(L_train[0].toarray())
```

Comment:
Why use dense format here? It will cause potential overhead?

Reply:
This is due to the API change at snorkel. They are now mainly use a dense format for matrices.

Reply:
To avoid the conversion (sparse to dense) overhead, `labeler.get_label_matrices` should return a dense format as it's been standardized at Snorkel like below:
> label matrix is dense outside of compute modules

Reply:
There is no reason to return dense matrix here since the label matrix supposes to be sparse.